workshop/4-Messaging/pinpoint-templates/abandoned-cart-email-template.html

```diff
@@ -1,27 +1,508 @@
+      }
+    }
+  </style>
 </head>
-<body>
-<center>
-    <table border="0" cellpadding="0" cellspacing="0">
-    <tr>
-        <td>
-            <p><strong>Hi {{User.UserAttributes.FirstName}}</strong>-
-            </p>
-            <p>We noticed that you recently put some items in your shopping cart but didn't complete your order. If there is anything we can do to help with your decision, please let us know. In addition, we'd like to offer you a $5 coupon for being a valued customer. Just enter ABC123 in the promotion code field at checkout to redeem your coupon. This coupon will exire in one week.
-            </p>
-            <p>Cheers,<br/>Retail Demo Store team
-            </p>
-        </td>
-    </tr>
-    <tr>
-        <td style="text-align: center; padding-top: 20px">
-            <small>Retail Demo Store &copy; 2019-2020</small>
-        </td>
-    </tr>
-    </table>
-</center>
+
+<body style="background-color:#d6dde5;">
+  <div style="background-color:#d6dde5;">
+    <!--[if mso | IE]>
+      <table
+         align="center" border="0" cellpadding="0" cellspacing="0" class="" style="width:600px;" width="600"
+      >
+        <tr>
+          <td style="line-height:0px;font-size:0px;mso-line-height-rule:exactly;">
+      <![endif]-->
+    <div style="background:#232F3E;background-color:#232F3E;margin:0px auto;max-width:600px;">
+      <table align="center" border="0" cellpadding="0" cellspacing="0" role="presentation" style="background:#232F3E;background-color:#232F3E;width:100%;">
+        <tbody>
+          <tr>
+            <td style="direction:ltr;font-size:0px;padding:20px 0;padding-bottom:0px;text-align:center;">
+              <!--[if mso | IE]>
+                  <table role="presentation" border="0" cellpadding="0" cellspacing="0">
+                
+        <tr>
+      
+            <td
+               class="" style="vertical-align:top;width:180px;"
+            >
+          <![endif]-->
+              <div class="mj-column-per-30 mj-outlook-group-fix" style="font-size:0px;text-align:left;direction:ltr;display:inline-block;vertical-align:top;width:100%;">
+                <table border="0" cellpadding="0" cellspacing="0" role="presentation" style="vertical-align:top;" width="100%">
+                  <tr>
+                    <td align="center" class="hide_on_mobile" style="font-size:0px;padding:10px 25px;padding-top:10px;padding-right:0px;padding-bottom:10px;padding-left:0px;word-break:break-word;">
+                      <table border="0" cellpadding="0" cellspacing="0" role="presentation" style="border-collapse:collapse;border-spacing:0px;">
+                        <tbody>
+                          <tr>
+                            <td style="width:62px;"> <img alt="Box free shipping" height="auto" src="{{Attributes.WebsiteURL}}/RDS_logo_white.svg" style="border:none;display:block;outline:none;text-decoration:none;height:auto;width:100%;font-size:13px;" width="62" />                              </td>
```

Comment:
Instead of using `Attributes.WebsiteURL` within this template, could we make it the front end's responsibility to send absolute URLs wherever needed? This isolates the magic values to one section of the code base.

Reply:
Updated!